src/applications/mhv/secure-messaging/components/AttachmentsList.jsx

```diff
@@ -232,19 +246,21 @@ const AttachmentsList = props => {
             </li>
           ))}
       </ul>
-      <RemoveAttachmentModal
-        visible={isModalVisible}
-        onClose={() => {
-          setIsModalVisible(false);
-          setIsAttachmentRemoved(false);
-        }}
-        onDelete={() => {
-          setNavigationError();
-          setIsModalVisible(false);
-          removeAttachment(fileToRemove);
-        }}
-        data-testid="remove-attachment-modal"
-      />
+      {(compose || reply) && (
```

Comment:
not substantial, but appears there is no difference in the use case of these 2 props, I would personally go with with 1 prop like `isEdit` or something. Unless we are planning to have separate conditions for each of these props.

Reply:
These props are used independently on separate pages, compose and reply screens. One or the other would have to be true for the modal to be used.

Reply:
right, but they end up both doing the same thing within the component. You can technically still send like `isEdit` prop from both pages without sacrificing any functionality. I guess it is just a matter of preference and keeping code less confusing